news/2025/2025-02-07-osumania-7k-world-cup-2025-quarterfinals-recap.md

```diff
@@ -0,0 +1,148 @@
+
+But even with that much neede context, our teams could not hit the brakes for even a split second. Despite Japan's strong rice players, Indonesia wasn't any less great at it. The same could be said the other way around. Japan almost learnt it the hard way when they picked [Silentroom - Aliake (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2314239#mania/4952522) (RC1) and Indonesia managed to almost even out their score with a 582-point difference. This forced Indonesia to pick their protect and get some ground back, bringing the score back up to 2–2.
+
+The rice versus long note picks continued all the way up to 3–3. Indonesia was out of LN maps to choose from, while Japan had plenty of options left. With Japan winning [69 de 74 - Grievous Fate](https://osu.ppy.sh/beatmapsets/2314385#mania/4952820) (RC6) and upping the lead to 4–3, Indonesia had no choice but to start picking the remaining hybrid maps. This ended up with Indonesia picking [PASOCOM MUSIC CLUB feat. punipunidenki - i s o p h o t e](https://osu.ppy.sh/beatmapsets/2314588#mania/4953343) (HB2), and a point that was very closely contested by ::{ flag=ID }:: [RaffCo](https://osu.ppy.sh/users/7589503) and ::{ flag=JP }:: [butanic](https://osu.ppy.sh/users/1433586)'s full combos given their teammates slip-ups. With only a 2,850-point difference, Japan managed to gain the upper hand, getting themselves in a very good position with both a break and match point with a 5–3 lead.
+
+Unfortunately, Indonesia was put between a rock and a hard place given Japan's upper hand in the pick. Despite ::{ flag=ID }:: [RaffCo](https://osu.ppy.sh/users/7589503) and ::{ flag=ID }:: [_yea](https://osu.ppy.sh/users/20834055)'s very commendable efforts, their doom was already spelled. And by winning [saaa - MORNINGLOOM](https://osu.ppy.sh/beatmapsets/2314251#mania/4952555) (RC4) Japan won the last point they needed to win the match! In the battle between this tournament's rice and long note titans, this year's skirmish was won by the former.
+
+Indonesia is not out of the game however! They'll get a second chance next weekend against the Russian Federation, in the hopes to climb their way back up to the Finals. Japan will not have such an easier time as they will be against South Korea. We hope both teams can perform as well as they did this weekend!
```

Comment:
```suggestion
Indonesia is not out of the game however! They'll get a second chance next weekend against the Russian Federation, in the hopes to climb their way back up to the Finals. At the same time, Japan will not have it any easier since they will be up against South Korea. We hope both teams can perform as well as they did this weekend!
```